flyteadmin/pkg/manager/impl/validation/execution_validator.go

```diff
@@ -131,7 +131,7 @@ func CheckValidExecutionID(executionID, fieldName string) error {
 	matched := executionIDRegex.MatchString(executionID)
 
 	if !matched {
-		return errors.NewFlyteAdminErrorf(codes.InvalidArgument, "invalid %s format: %s", fieldName, executionID)
+		return errors.NewFlyteAdminErrorf(codes.InvalidArgument, "invalid %s format: %s, does not match regex '^[a-z][a-z\-0-9]*$'", fieldName, executionID)
```

Comment:
can we not copy paste the regex, instead print the regex value itself.
Also, update the unit tests